doc/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
It might be good to add a note somewhere (README?) saying "These files were generated with `sphinx-quickstart` as of version X.Y.Z".

Sphinx occasionally updates, e.g. these files are now a lot shorter than they used to be. Noting the version can prompt users to think whether they'd need to refresh the files to work more seamlessly with updated versions of Sphinx.

Reply:
done